
# This is a patch for DBD-mysql-4.019 to update it to DBD-mysql-4.019w
# 
# To apply this patch:
# STEP 1: Chdir to the source directory.
# STEP 2: Run the 'applypatch' program with this patch file as input.
#
# If you do not have 'applypatch', it is part of the 'makepatch' package
# that you can fetch from the Comprehensive Perl Archive Network:
# http://www.perl.com/CPAN/authors/Johan_Vromans/makepatch-x.y.tar.gz
# In the above URL, 'x' should be 2 or higher.
#
# To apply this patch without the use of 'applypatch':
# STEP 1: Chdir to the source directory.
# STEP 2: Run the 'patch' program with this file as input.
#
#### End of Preamble ####

#### Patch data follows ####
diff -c "DBD-mysql-4.019\dbdimp.h" "DBD-mysql-4.019w\dbdimp.h"
Index: ./dbdimp.h
*** ./dbdimp.h	Mon May  9 04:11:37 2011
--- ./dbdimp.h	Mon Jul 18 03:17:23 2011
***************
*** 27,33 ****
  /* For now, we hardcode this, but in the future,
   * we can detect capabilities of the MySQL libraries
   * we're talking to */
! #define MYSQL_ASYNC 1
  
  /*
   * This is the version of MySQL wherer
--- 27,33 ----
  /* For now, we hardcode this, but in the future,
   * we can detect capabilities of the MySQL libraries
   * we're talking to */
! #define MYSQL_ASYNC 0
  
  /*
   * This is the version of MySQL wherer
#### End of Patch data ####

#### ApplyPatch data follows ####
# Data version        : 1.0
# Date generated      : Mon Jul 18 03:18:00 2011
# Generated by        : makepatch 2.04
# Recurse directories : Yes
# Excluded files      : (\A|/).*\~\Z
#                       (\A|/).*\.a\Z
#                       (\A|/).*\.bak\Z
#                       (\A|/).*\.BAK\Z
#                       (\A|/).*\.elc\Z
#                       (\A|/).*\.exe\Z
#                       (\A|/).*\.gz\Z
#                       (\A|/).*\.ln\Z
#                       (\A|/).*\.o\Z
#                       (\A|/).*\.obj\Z
#                       (\A|/).*\.olb\Z
#                       (\A|/).*\.old\Z
#                       (\A|/).*\.orig\Z
#                       (\A|/).*\.rej\Z
#                       (\A|/).*\.so\Z
#                       (\A|/).*\.Z\Z
#                       (\A|/)\.del\-.*\Z
#                       (\A|/)\.make\.state\Z
#                       (\A|/)\.nse_depinfo\Z
#                       (\A|/)core\Z
#                       (\A|/)tags\Z
#                       (\A|/)TAGS\Z
# p "dbdimp.h" 10485 1310948243 0100666
#### End of ApplyPatch data ####

#### End of Patch kit [created: Mon Jul 18 03:18:00 2011] ####
#### Patch checksum: 55 1879 48809 ####
#### Checksum: 73 2571 40378 ####
